<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMode</class>
 <widget class="QDialog" name="UserMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>921</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="FindEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>165</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>查 找</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BorrowButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>借阅</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>还书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBackButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FastReturnButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>快速还书</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ReturnBookEdit">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>470</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>470</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>登记号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MyInfo">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>我的信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
